pass-manager-svc/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  application:
+    name: PassManagingApplication
+  data:
+    mongodb:
+      uri: mongodb://root:test@localhost:27018/test_db?authSource=admin
+nats:
+  uri: nats://localhost:4223
+
+
+mongock:
+  enabled: false
```

Comment:
this approach may lead to a situation when you're running your integration tests with not configured DB (e.g. absence of unique indexes may lead to document duplication when uniqueness is expected)